app/routes/user/password.tsx

```diff
@@ -54,6 +54,18 @@ export function WrongIdp() {
   )
 }
 
+export function ErrorMessage(errorMessage: any, interacted: boolean) {
```

Comment:
No need to export this function:

```suggestion
function ErrorMessage(errorMessage: any, interacted: boolean) {
```